assets/dev/scss/frontend/_container.scss

```diff
@@ -62,8 +79,8 @@
 	&-full,
 	& > .e-con-inner {
 		text-align: var(--text-align);
-		padding-block-start: var(--padding-top, var(--padding-block-start));
-		padding-block-end: var(--padding-bottom, var(--padding-block-end));
+		padding-block-start: var(--bc-padding-block-start, var(--padding-block-start));
```

Comment:
Would this not make it more confusing to know when we're dealing with directional props?